content/basics/basics.mdx

```diff
@@ -0,0 +1,327 @@
+---
+name: Basics
+route: /basics
+---
+
+import { Playground } from 'docz'
+
+# Basic Introduction
+
+[MobX](https://mobx.js.org/README.html) lets you create observable state. This
```

Comment:
Link to the site directly in case we would restructure it in the future.
```suggestion
[MobX](https://mobx.js.org) lets you create observable state. This
```